tests/test_host_check.py

```diff
@@ -650,7 +815,42 @@ def test_one_plat_supported(self, capsys):
 ============================================================================
 """
         assert output == cli_output
-        assert exit_code == 0
+        assert exit_code == EXIT_ERROR
+
+    def test_one_plat_failing_other_warning(self, capsys):
+        """Test only one platform failing and the other warning."""
+        exit_code, output = self.run_host_check(
+            capsys,
+            [],
+            failing_checks=["Hugepages"],
+            warning_checks=["Kernel version"],
+        )
+        cli_output = f"""\
+==============================
+Platform checks
+==============================
+
+base checks
+-----------------------
+Checks: {BASE_CHECKS_STR}
+
+xrd-control-plane checks
+-----------------------
+Checks: {CONTROL_PLANE_CHECKS_STR}
+
+xrd-vrouter checks
+-----------------------
+Checks: {VROUTER_CHECKS_STR}
+
+============================================================================
+XR platforms NOT supported: xrd-vrouter
+!! One or more platform checks resulted in a warning for XR platforms:    !!
+      xrd-control-plane
+   See warnings above.
```

Comment:
As above - I think !! around "XR platforms NOT supported: xrd-vrouter" is very important here.  Currently this looks like the warning for xrd-control-plane is the most impactful thing to report to the user.  I think the user could very easily misread this and think that xrd-vrouter is supported.